actions/project.go

```diff
@@ -52,9 +53,9 @@ func DownloadTemplate(c *cli.Context) {
 
 	// Remove invalid characters from the string we will use
 	// as the project name in the template.
-	r := regexp.MustCompile("[^a-zA-Z0-9._-]");
```

Comment:
Good spot!

Reply:
It was actually the auto-linter that changed this :)